pages/03.installation/01.environment/04.lando/docs.md

```diff
@@ -0,0 +1,188 @@
+---
+---
+title: Lando
+metadata:
+  description: Lando is a wrapper around Docker that simplifies the process for PHP applications to run on Docker.
+taxonomy:
+  category: docs
+---
+
+>>> To contribute to this documentation, please submit a pull request to our [learn repository](https://github.com/userfrosting/learn/tree/master/pages).
+
+Lando provides a stable and easy-to-use local development environment. It allows you to integrate PHPMyAdmin and most PHP IDE debuggers hook simply into this as well.
+
+>> Note: Lando is not meant for production!
+
+>> Note: Lando works best with Mac & Linux, but does support Windows as well. Note that if you are using Windows, you must enable Hyper-V. Doing so prevents you from running Vagrant/VMBox. If you are able to run Docker on Windows, you _should_ be able to run Lando on Windows as well.
+
+## Installation Steps
+
+### Install & Configure Lando
+
+Start by installing [Lando](https://docs.lando.dev/basics/installation.html#system-requirements).
+
+Next, create your Lando configuraton file in your application root. Call the file `.lando.yml`. A sample configuration is provided below:
+
+#### Sample Lando Configuration File
+
+```yml
```

Comment:
Since we are already including helper services (PHP My Admin) we might as well go all the way. Lando supports the MailHog service (https://docs.lando.dev/config/mailhog.html) which would be a fantastic addition.

Reply:
Seems some work will need to be done on UserFrosting first to make using the MailHog service easier to integrate with.